src/chartGallery.tsx

```diff
@@ -94,7 +95,7 @@ class ChartGalleryComponent extends Component<chartGalleryProps,any> {
                       selected={true} 
                       onClick={(e) => {this.onItemSelected(idx);}}>
                       {'vislib' in item && 'config' in item && JSON.stringify(item['vislib']).substring(1, JSON.stringify(item['vislib']).length - 1) === 'matplotlib' ?
-                      <img id="gal-img" src={"data:image/png;base64," + JSON.stringify(item['config']).substring(1, JSON.stringify(item['config']).length - 1) + "\ "}></img> :
+                      <img id="gal-img" src={"data:image/png;base64," + JSON.stringify(item['config']).substring(1, JSON.stringify(item['config']).length - 1) + "\ "} style={{ height: `${50 + 160 * this.props.plottingScale}px`}}></img> :
```

Comment:
Do we need the plotting scale here, or could we simply pre-render these as larger charts in the backend code?

Reply:
I tried making them larger in the backend code in `utils.py#matplotlib_setup` with `fig, ax = plt.subplots(figsize=(w * plotting_scale, h * plotting_scale))`. Upon doing this, the Matplotlib graph gets more fine grained without actually increasing in size. Do you have any pointers on this?

Reply:
Yeah I think the figsize just increases the size of the canvas without changing other things like the title or label font size. There might be other approaches to scale up/down the matplotlib chart more uniformly, similar to what happens in Altair. It's worth looking into if this is possible.

Reply:
Currently, I don't have a great resolution to this. I've tried multiple cases including modifying figure size upon saving the file in `MatplotlibRenderer` and other approaches in `utils.py#matplotlib_setup`. I think part of the problem is that we're saving the chart as an image rather than passing in the chart directly like in Altair. I'm happy to continue trying out more approaches, but I'd argue that it may be out of scope for this PR as it may require some more drastic changes on the Matplotlib front.

Reply:
Thanks @micahtyong, let's defer the matplotlib figure sizing problem to another issue, given that changing the svg approach currently would require some additional work. Could you open an issue for Matplotlib sizing so that we can look into this in the future?